### DISCUSSION TITLE
Keyball44 左側（トラックボール側）のOLED含む全てのキーが反応しない。

### DISCUSSION THREAD

**Discussion by tetraohno** [Yowkees/keyball#300] [Q&A] [answered]
今回初めて自作キーボードを購入しました。

Keyball44の左側（トラックボール側）のキーが全て反応せず、OLEDも表示されない状態で困ってます。
半田付けは全て行い、一度はOLEDが点灯したのですが、一部キーが入力できなかったため、はんだの量が少ないところを再度半田付けを行いましたところ、OLED、キーが使用できなくなってしまいました。

どのようにしたら良いかご教授ください。
![IMG_1518](https://github.com/Yowkees/keyball/assets/54048437/7251da0a-82ed-4f71-8788-8219b4631ebe)
![IMG_1519](https://github.com/Yowkees/keyball/assets/54048437/44d09125-8eb5-4ff8-8377-483a36bb014e)
![IMG_1520](https://github.com/Yowkees/keyball/assets/54048437/47b39900-4652-4dfc-a33f-f17f5edfd4d3)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

ProMicroを差し込む位置が違います。基板の表面から見て、黒枠内にコンスルーが入るようにProMicroを差し込んでください。
お手数ですがご対応お願いいたします。

**Reply by tetraohno** [reply]
ありがとうございます。盲点でした。
